ruby/hamming/hamming.rb

```diff
@@ -0,0 +1,19 @@
+class Hamming
+  def self.compute(string1, string2)
+    var = 0
+    if string1.size == string2.size
+      string1.each_char.with_index do |element, index|
+        string2.each_char.with_index do |e, i|
```

Comment:
Let's try removing the inner loop. What if we try checking both strings at the same time?

Reply:
Yes I think so, I was looking into the best way to do this and I think that by splitting one of the arrays I can achieve this. Additionally, I think a guard clause can be included in this method too.